src/renderer/components/playlist-info/playlist-info.js

```diff
@@ -99,12 +99,18 @@ export default Vue.extend({
 
       switch (method) {
         case 'copyYoutube':
+          this.showToast({
+            message: this.$t('Share.YouTube URL copied to clipboard')
+          })
           navigator.clipboard.writeText(youtubeUrl)
```

Comment:
Maybe move the toast to after the clipboard gets written to?

Reply:
Sure, makes sense.